### DISCUSSION TITLE
How to Set PlaceHolder Text Color

### DISCUSSION THREAD

**Discussion by Orgbrat** [dotnet/maui#5266] [Q&A]
So I see there is a property on the Entry Control to set PlaceHolderColor.  But it will not compile as the compiler says missing property. After researching it looks like the MAUI team has moved this property to .NET 7. So is there a way to set the placeholder Text Color at this time?

Orgbrat

**Comment by Pushpalatha12-eng** [comment]
Still the PlaceholderColor does not work.

**Reply by FM1973** [reply]
Using a style works for me.

**Reply by Pushpalatha12-eng** [reply]
Thank you. Will try this.
